### PR TITLE
[Merged by Bors] - remove API to start syncing

### PR DESCRIPTION
**Pull request by countvonzero**
## Motivation
part of #4140

## Changes
don't allow API to start syncing when the node is not ready (state not loaded...etc)

### CODE REVIEW

File: api/grpcserver/node_service.go [hunk 1]
```diff
@@ -111,11 +111,8 @@ func (s NodeService) getLayers() (curLayer, latestLayer, verifiedLayer uint32) {
 
 // SyncStart requests that the node start syncing the mesh (if it isn't already syncing).
 func (s NodeService) SyncStart(context.Context, *pb.SyncStartRequest) (*pb.SyncStartResponse, error) {
-	log.Info("GRPC NodeService.SyncStart")
-	s.syncer.Start(s.appCtx)
-	return &pb.SyncStartResponse{
-		Status: &rpcstatus.Status{Code: int32(code.Code_OK)},
-	}, nil
+	log.Info("UNIMPLEMENTED GRPC NodeService.SyncStart")
```
Comment (dshulyak) [inline api/grpcserver/node_service.go]: i think this can be removed, there is a grpc middleware that logs errors

**Review by dshulyak — APPROVED** [review]

**Comment by countvonzero** [comment]
bors merge

**Comment by pigmej** [comment]
@kacpersaw we need to change explorer and dash to adopt to that and then we merge this :)

**Comment by countvonzero** [comment]
> @kacpersaw we need to change explorer and dash to adopt to that and then we merge this :)

is it strictly necessary tho? you can make the cut for devnet at the PR before this one...

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**